### DISCUSSION TITLE
"Failed to replace env in config: ${GITHUB_ACCESS_TOKEN}"

### DISCUSSION THREAD

**Discussion by k-rajat19** [razorpay/blade#820] [Q&A] [answered]
Hi,
I'm getting  this error while running [yarn install](https://github.com/razorpay/blade/blob/master/CONTRIBUTING.md?plain=1#L14)
```
error An unexpected error occurred: "Failed to replace env in config: ${GITHUB_ACCESS_TOKEN}".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\dell\\Desktop\\blade\\packages\\blade\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
any suggestions on what should I need to do?

**Comment by divyanshu013** [comment]
Could you please try the following steps and then check? Earlier we were publishing Blade only to GitHub registry hence these steps. I suppose these can be amended now that we're also publishing to npm /cc @kamleshchandnani 

* You must be running Node version >=14.0.0
* You must have `yarn` installed
* Generate a Personal Access Token on GitHub by [visiting this link](https://github.com/settings/tokens/new?scopes=repo,workflow,write:packages,read:repo_hook,write:packages)
  - If you belong to razorpay organisation then you will need to enable SSO by clicking `Authorize` button next to Razorpay logo.
* Run `code ~/.bashrc` or `code ~/.zshrc` in your editor and add this line 
  ```
  export GITHUB_ACCESS_TOKEN="<YOUR_TOKEN>"
  ```
  > Note: Replace `<YOUR_TOKEN>` with your actual GitHub Personal Access Token
* Run `source ~/.bashrc` or `source ~/.zshrc` based on the file you added your token.
* Run `code ~/.npmrc` and append the following
```bash
# add following to your .npmrc
@razorpay:registry=https://npm.pkg.github.com/
//npm.pkg.github.com/:always-auth=true
//npm.pkg.github.com/:_authToken=${GITHUB_ACCESS_TOKEN}
```

**Comment by kamleshchandnani** [comment, marked as answer]
can you just remove the `.npmrc` file and do `yarn` install again? previously we were publishing to GitHub registry for which you were required to have `GITHUB_ACCESS_TOKEN` but now we are publishing it to global npm registry so you don't need `.npmrc`
